src/libraries/System.IO.Compression/src/System/IO/Compression/TarFile.cs

```
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Compression
+{
+    public class TarFile
+    {
+        public static TarArchive OpenRead(string archiveFileName) => Open(archiveFileName, TarArchiveMode.Read);
+
+        public static TarArchive Open(string archiveFileName, TarArchiveMode mode)
+        {
+            if (string.IsNullOrEmpty(archiveFileName))
+            {
+                throw new ArgumentNullException(nameof(archiveFileName));
```

Consider throwing `ArgumentException` instead of `ArgumentNullException` if you're going to handle empty strings as part of the same check.